code/modules/hydroponics/grown/berries.dm

```diff
@@ -43,11 +44,12 @@
 /obj/item/reagent_containers/food/snacks/grown/berries/poison
 	seed = /obj/item/seeds/berry/poison
 	name = "bunch of poison-berries"
-	desc = "Taste so good, you could die!"
+	desc = "Taste so good, you might die!"
```

Comment:
don't ruin the joke

Reply:
>told to not ruin the joke
>changes it
>still ruins the joke

Reply:
these are two different berries @Alek2ander

Reply:
@JJRcop oh

also you pinged the wrong guy

Reply:
maybe he'll come back!